native/cocos/renderer/gfx-base/GFXDef-common.h

```diff
@@ -1401,7 +1405,9 @@ struct ALIGNAS(8) BufferBarrierInfo {
 
     Queue *srcQueue{nullptr}; // @ts-nullable
     Queue *dstQueue{nullptr}; // @ts-nullable
-
+#if CC_CPU_ARCH == CC_CPU_ARCH_32
+    uint32_t _padding{0};
```

Comment:
`uint32_t _padding` might be added between `size` and `discardContents` without macro.

Reply:
I will check that